roboswag/cli.py

```diff
@@ -48,6 +65,14 @@ def cli():
     metavar="OUTPUT_DIR",
     help="Output directory path",
 )
-def generate(spec: str, output_dir: Optional[Path] = None):
+@click.option(
+    "-a",
+    "--authorization",
```

Comment:
What about just `--auth` for a long name? It would also drop the risk of someone mixing up 'authorization' and 'authentication'.